CHANGELOG.rst

```diff
@@ -1,6 +1,30 @@
 Change log
 ==========
 
+0.19.1 (2020-11-06)
+-------------------
+
+New features
+~~~~~~~~~~~~
+
+* **config:** add classes :class:`~universum.configuration_support.Step` and
+  :class:`~universum.configuration_support.Configuration` to :doc:`configuration support <configuration_support>`
+  for type checking and increased usability
+* **config:** default config name ``.universum.py`` is :doc:`now supported <configuring>` along with
```

Comment:
I would say --config is now not mandatory. If it is not specified, the default value is used.